adopters.md

```diff
@@ -7,31 +7,71 @@ category: adopters
+|[GAEA](http://www.gaea.com/en/)|Gaming|[Chinese](https://www.pingcap.com/cases-cn/user-case-gaea-ad/)|[English](https://www.pingcap.com/blog/2017-05-22-Comparison-between-MySQL-and-TiDB-with-tens-of-millions-of-data-per-day/)|
+|[YOOZOO Games](https://www.crunchbase.com/organization/yoozoo-games)|Gaming|[Chinese](https://pingcap.com/cases-cn/user-case-youzu/)||
+|[Seasun Games](https://www.crunchbase.com/organization/seasun)|Gaming|[Chinese](https://pingcap.com/cases-cn/user-case-xishanju/)||
+|[NetEase Games](https://game.163.com/en/)|Gaming|||
+|[FUNYOURS JAPAN](http://company.funyours.co.jp/)|Gaming|[Chinese](https://pingcap.com/cases-cn/user-case-funyours-japan/)||
+|[Zhaopin.com](https://www.crunchbase.com/organization/zhaopin)|Recruiting|||
+|[Panda.tv](https://www.crunchbase.com/organization/panda-tv)|Live Streaming|||
+|[Hoodinn](https://www.crunchbase.com/organization/hoodinn)|Gaming|||
+|[Ping++](https://www.crunchbase.com/organization/ping-5)|Mobile Payment|[Chinese](https://pingcap.com/cases-cn/user-case-ping++/)||
+|[Hainan eKing Technology](https://www.crunchbase.com/organization/hainan-eking-technology)|Enterprise Technology|[Chinese](https://pingcap.com/cases-cn/user-case-ekingtech/)||
+|LianLian Tech|Mobile Payment|||
+|[Tongdun Technology](https://www.crunchbase.com/organization/tongdun-technology)|FinTech|||
+|[Wacai](https://www.crunchbase.com/organization/wacai)|FinTech|||
+|Tree Finance|FinTech|||
+|2Dfire.com|FoodTech|[Chinese](https://www.pingcap.com/cases-cn/user-case-erweihuo/)||
+|[Happigo.com](https://www.crunchbase.com/organization/happigo-com)|E-commerce|||
+|[Mashang Consumer Finance](https://www.crunchbase.com/organization/ms-finance)|FinTech|||
+|[Tencent OMG](https://en.wikipedia.org/wiki/Tencent)|Media|||
+|[Terren](http://webterren.com.zigstat.com/)|Media|||
+|[LeCloud](https://www.crunchbase.com/organization/letv-2)|Media|||
+|[Miaopai](https://en.wikipedia.org/wiki/Miaopai)|Media|||
+|[Snowball Finance](https://www.crunchbase.com/organization/snowball-finance)|FinTech|||
+|[Yimutian](http://www.ymt.com/)|E-commerce|||
+|[Gengmei](https://www.crunchbase.com/organization/gengmei)|Plastic Surgery|||
+|[Acewill](https://www.crunchbase.com/organization/acewill)|FoodTech|||
+|[Keruyun ](https://www.crunchbase.com/organization/keruyun-technology-beijing-co-ltd)|SaaS|[Chinese](https://pingcap.com/cases-cn/user-case-keruyun/)||
+|[Youju Tech](https://www.ujuz.cn/)|E-Commerce|||
+|[Maizhuo](https://www.crunchbase.com/organization/maizhuo)|E-Commerce|||
+|[Mogujie](https://www.crunchbase.com/organization/mogujie)|E-Commerce|||
+|[Zhuan Zhuan](https://www.crunchbase.com/organization/zhuan-zhuan)|Online Marketplace|[Chinese](https://pingcap.com/cases-cn/user-case-zhuanzhuan/)||
+|[Shuangchuang Huipu](http://scphjt.com/)|FinTech|||
+|[Meizu](https://en.wikipedia.org/wiki/Meizu)|Media||
+|[Seagroup](https://sea-group.org/?lang=en)|Gaming|||
```

Comment:
For the company name, they use "SEA group" on their official website: https://sea-group.org/?cat=1&lang=en